hawkbit-security-core/src/main/java/org/eclipse/hawkbit/security/InMemoryUserAuthoritiesResolver.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2020 Bosch.IO GmbH and others.
```

Comment:
I guess you forgot the empty line after Copyright (compare with UserAuthoritiesResolver.java) ;)